### DISCUSSION TITLE
Screenshot: add features

### DISCUSSION THREAD

**Discussion by simotro** [linuxmint/discussions#492] [Ideas]
It would be very interesting and productive if some features like Flameshot were added. What do you think?

**Comment by klangman** [comment]
Wow.. Flameshot is awesome, I'll be using it from now on!

Question, why should these advanced features be added to Mint?? Flameshot seems to work fine for those that need the features.

**Reply by simotro** [reply]
To have these amazing features out of the box!! ;)

**Reply by ghost** [reply]
I second this, 100% agree - I prefer feature richness right out of the box, and while Cinnamon is ok, there are many features missing, that a sane person would miss. Thanks for this btw, I never heard of Flameshot, but finally you helped me to find a great alternative to KDE's Spectacle (which I friggin loved) - I rly needed the instant arrowing, and pixelating stuffs. Thank you man, and I upvoted your idea, hope they make it the default, instead of this flhasbang gnome-screenshot already lol

**Comment by mtwebster** [comment]
Flameshot is nice - I use it occasionally, but it's exclusively for area screenshots - and lacks a simple UI for novice users.

- no fullscreen screenshot (without using a shortcut)
- no active-window screenshot
- no keyboard shortcuts (other than those in-app)
- complicated configuration
- no configurable delay

**Reply by mtwebster** [reply]
It was pointed out to me you can configure a delay (and fullscreen) using the 'capture launcher'. This sort of adds to my point of a 'simple ui/easy configuration, though.

**Reply by simotro** [reply]
It might make sense to simply add the ability to arrow, mark, blur, and write to screenshots taken with Mint's standard system.

**Comment by Jopp-gh** [comment]
Flameshot is great, maybe a bit too much fora default app.

I would be happy already if Mint's default screenshot app would be able to draw some colored rectangles, ellipses, lines, maybe add some text.

**Comment by goyalyashpal** [comment]
duplicate of 
* #524
